corehq/apps/case_importer/views.py

```diff
@@ -380,9 +381,59 @@ def excel_fields(request, domain):
     return render(request, "case_importer/excel_fields.html", context)
 
 
+def find_outlier_column(spreadsheet, col_data_counts, outlier_col, sheet_name, max_z_score):
+    Z_SCORE_THRESHOLD = 1.5
+    mean = statistics.mean(col_data_counts)
+    std_dev = statistics.pstdev(col_data_counts)
```

Comment:
The nerd in me wants to say it so I'll just say it. Probably want sample standard deviation `statistics.stdev` here.